src/Core/src/Handlers/SearchBar/SearchBarHandler.Android.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Android.Widget;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class SearchBarHandler : AbstractViewHandler<ISearchBar, SearchView>
```

Comment:
I think we should use the AppCompat `SearchView` instead.

Reply:
Hmm, I just took the Entry one. Should that not then also use `using AndroidX.AppCompat.Widget;`?

Reply:
Fixed

Reply:
We probably should. I'll open an issue: https://github.com/dotnet/maui/issues/466